constants/rateLimtingMiddelware.js

```diff
@@ -0,0 +1,23 @@
+const RATE_LIMITER_SLOW_BRUTE_BY_IP_OPTIONS = {
+  points: 100,
+  duration: 60 * 60 * 24,
+  blockDuration: 60 * 60 * 24,
+  keyPrefix: "login_fail_by_ip_per_day",
+};
```

Comment:
Instead of blocking the users for a day,  1/2 hour or 1 hour should be sufficient. But still check with @ankushdharkar once.

Reply:
I will check limit and block duration.